### PR TITLE
Fix: Delete 'inprogress' ImageProcessingStatus Rows when exception occurs

### PR DESCRIPTION
**Pull request by chrise96**
BCV-592

### CODE REVIEW

File: yolov5/baas_utils/error_handling.py [hunk 1]
```diff
@@ -54,6 +55,33 @@ def wrapper(*args, **kwargs):
...
+                    session.query(ImageProcessingStatus).filter(
+                        and_(
+                            ImageProcessingStatus.image_customer_name == subquery.c.image_customer_name,
+                            ImageProcessingStatus.image_upload_date == subquery.c.image_upload_date,
+                            ImageProcessingStatus.image_filename == subquery.c.image_filename
+                        )
+                    ).delete(synchronize_session=False)
+
+                    LOGGER.info(f"Deleted 'inprogress' rows in ImageProcessingStatus for run_id: {run_id}")
+
```
Comment (epureanudiana) [inline yolov5/baas_utils/error_handling.py:84]: we said that we want to store the errors in the database. Are we doing this now?

Comment (epureanudiana) [reply yolov5/baas_utils/error_handling.py:84]: In this case it could be that the images are corrupted, but where do we store this info about the error?

Comment (chrise96) [reply yolov5/baas_utils/error_handling.py:84]: We store this in BatchRunInformation, this is some lines below your comment. It takes the error code of  "except Exception as e:"

Comment (epureanudiana) [reply yolov5/baas_utils/error_handling.py:84]: alright, nice!

**Review by epureanudiana — APPROVED** [review]

**Pull request merged**